build.sbt

```diff
@@ -2,41 +2,39 @@ import sbt.Keys._
 import sbt._
 import sbt.Tests._
 
-
 val defaultSettings = Defaults.coreDefaultSettings ++ xerial.sbt.Sonatype.sonatypeSettings ++ Seq(
-  organization := "com.github.spinalhdl",
-  version      := SpinalVersion.all,
-  crossScalaVersions := SpinalVersion.compilers,
-  scalaVersion := SpinalVersion.compilers(0),
-  scalacOptions ++= Seq("-unchecked","-target:jvm-1.8"/*, "-feature" ,"-deprecation"*/),
+  organization          := "com.github.spinalhdl",
+  version               := SpinalVersion.all,
+  crossScalaVersions    := SpinalVersion.compilers,
+  scalaVersion          := SpinalVersion.compilers(0),
```

Comment:
This automatic allignement turn me on <3

Reply:
Please don't. I once had a project with alignment enabled (because it honestly looks cool), but it quickly gets seriously annoying with the diffs. Each time a parameter with a longer name gets introduced or removed on the left side, the entire right side needs to be readjusted. When one entry on the left side is a lot longer than all the others, this results in much ugly whitespace too. There were also issues with multiline entries, but I think those might have been Java specific.

Reply:
Whitespace diffs are very easy to ignore with modern diff tools, but readability of the code is much better with alignment.  Perhaps for super critical sections there is a way to turn formatter off in the code ?

Reply:
>  but it quickly gets seriously annoying with the diffs

That make a lot of sense.

Thanks for the feedbacks ^^
Let's turn it of then.